modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -270,12 +266,13 @@ public void processOpts() {
             Set<String> modelImports = new HashSet<>();
             CodegenModel cm = (CodegenModel) mo.get("model");
             for (String modelImport : cm.imports) {
-                if (importMapping.containsKey(modelImport)) {
-                    modelImports.add(importMapping.get(modelImport));
-                } else {
-                    if (!modelToIgnore.contains(modelImport.toLowerCase(Locale.ROOT))) {
-                        modelImports.add("package:" + pubName + "/model/" + underscore(modelImport) + ".dart");
+                if (importMapping().containsKey(modelImport)) {
+                    final String value = importMapping().get(modelImport);
+                    if (value != null && !"dart:core".equals(value)) {
```

Comment:
for simpliify `Objects.equals()` which gives you null-safety and fewer keystrokes

Reply:
Writing more Dart and Kotlin than Java lately, thanks!